package.json

```diff
@@ -33,26 +34,28 @@
   "scripts": {
     "dev": "start-storybook -p 6006",
     "build": "rm -fr ./dist && rollup -c",
-    "test": "npm run lint && npm run testonly && npm run flow",
-    "test:watch": "npm run testonly -- --watch --watch-extensions js",
+    "test": "npm run lint && npm run testonly && npm run tsc",
+    "test:watch": "npm run testonly -- --watch --watch-extensions ts",
     "testonly": "cross-env NODE_ENV=test mocha $npm_package_options_mocha",
     "release:major": "npm run build && npm version major",
     "release:minor": "npm run build && npm version minor",
     "release:patch": "npm run build && npm version patch",
     "lint": "eslint 'src/**/*.js'",
-    "flow": "flow",
+    "watch": "node_modules/.bin/tsc --watch",
```

Comment:
Missing `tslint` dependency and script. 
Once added the `tslint` you can remove `eslint` stuff